packages/twenty-server/src/engine/workspace-manager/workspace-cleaner/commands/destroy-workspace.command.ts

```diff
@@ -0,0 +1,48 @@
+import { Command, Option } from 'nest-commander';
+
+import {
+  MigrationCommandOptions,
+  MigrationCommandRunner,
+} from 'src/database/commands/command-runners/migration.command-runner';
+import { CleanerWorkspaceService } from 'src/engine/workspace-manager/workspace-cleaner/services/cleaner.workspace-service';
+
+@Command({
+  name: 'workspace:destroy',
+  description: 'Destroy workspace',
+})
+export class DestroyWorkspaceCommand extends MigrationCommandRunner {
+  private workspaceIds: string[] = [];
+
+  constructor(
+    private readonly cleanerWorkspaceService: CleanerWorkspaceService,
+  ) {
+    super();
+  }
+
+  @Option({
+    flags: '-w, --workspace-id <workspace_id>',
+    description: 'workspace id - mandatory',
+    required: true,
+  })
+  parseWorkspaceId(val: string): string[] {
```

Comment:
is it a singular or a plural here ? jsut curious since I guess it's a function inherited from the MigrationCommandRunner but it takes one worksapce as arguments and returns an array of workspace. I totally see this is working, it's just a bit strange at first